src/core/components/hv-screen/index.js

```diff
@@ -291,29 +298,37 @@ export default class HvScreen extends React.Component {
     }
   }
 
+  /**
+   * Implement the callbacks from this class
+   */
+  updateCallbacks = () => {
```

Comment:
I think this does not need to be a function, it can just be defined as a property
```suggestion
  updateCallbacks = {
```

Reply:
Updated